src/client/components/app/Navbar/index.tsx

```diff
@@ -36,7 +36,7 @@ const StyledText = styled.h1<{ toneDown?: boolean }>`
   ${({ toneDown, theme }) =>
     toneDown &&
     `
-    color: ${theme.colors.texts};
+    color: ${theme.colors.textsVariant};
```

Comment:
Dark before:
![image](https://user-images.githubusercontent.com/78362532/167502293-30429f6d-28a3-4c0d-b621-fe36645b06b9.png)

Dark now:
![image](https://user-images.githubusercontent.com/78362532/167502309-0272f48f-01b4-4d7c-8693-d6b8fd642355.png)

Is this change intended?

Reply:
no - will fix ty

Reply:
<img width="705" alt="image" src="https://user-images.githubusercontent.com/80323528/167607171-67288738-02a6-4053-9647-f8e9fc53f212.png">